docs/products/kafka/howto/kafka-tiered-storage-get-started.rst

```diff
@@ -0,0 +1,37 @@
+
+Get started with tiered storage for Aiven for Apache Kafka® 
+====================================================================
+
+Aiven for Apache Kafka®'s tiered storage expands storage beyond local disks. It stores frequently accessed data on faster tiers and less active data on cost-effective, slower tiers, ensuring both performance and cost efficiency are optimized.
+
+For an in-depth understanding of tiered storage, how it works, and its benefits, see `Tiered Storage in Aiven for Apache Kafka®`.
+
+.. important:: 
+    
+   Aiven for Apache Kafka® tiered storage is an early availability feature, which means it has some restrictions on the functionality and service level agreement. It is intended for non-production environments, but you can test it with production-like workloads to assess the performance. To enable this feature, navigate to the :doc:`Feature preview </docs/platform/howto/feature-preview>` page within your user profile.
```

Comment:
> To enable this feature, navigate to the :doc:`Feature preview </docs/platform/howto/feature-preview>` page within your user profile.

Hm.. how does this work? I don't think customers need enabling anything for TS to start working.

Reply:
I assumed that because this is an Early Availability feature, it would also be included as a Feature Preview. I believe @roope-kar  was handling the preview aspect — could you confirm this? If this isn't included in the Feature Preview and doesn't require activation, I'll make sure to revise the note accordingly.

(https://docs.aiven.io/docs/platform/concepts/beta_services#early-availability-public-beta https://docs.aiven.io/docs/platform/howto/feature-preview

Reply:
It's a console thing. By policy, Beta features should be placed under "Feature preview" UI. It controls the visibility of tiered storage stuff. It's located in Profile -> Feature preview. Here you can see list of new Beta features. Once enabled, you'll see related things in UI. It also provides means to provide feedback.

Ref: https://app.slack.com/client/E02TH02S2F3/C053VUDS28Z
Ref: https://docs.aiven.io/docs/platform/concepts/beta_services#early-availability-public-beta

Reply:
Here's a [screencast](https://github-production-user-asset-6210df.s3.amazonaws.com/15416578/272187738-8de4f34e-0c5a-4818-b6f2-4bd27244b564.webm?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=AKIAIWNJYAX4CSVEH53A%2F20231103%2Fus-east-1%2Fs3%2Faws4_request&X-Amz-Date=20231103T070102Z&X-Amz-Expires=300&X-Amz-Signature=221327002a60154f4df5094d2626367aafe09b4f75de0afde333820cb9f9bc93&X-Amz-SignedHeaders=host&actor_id=15416578&key_id=0&repo_id=29294097) if you want to see it in action.

Reply:
TIL!